Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -870,19 +870,35 @@ class BrowserViewController: UIViewController {
     }
 
     func finishEditingAndSubmit(_ url: URL, visitType: VisitType) {
-        topToolbar.currentURL = url
-        topToolbar.leaveOverlayMode()
+        let javascriptPrefix = "javascript:"
```

Comment:
is the URL normalized at this point? for instance `JAVASCRIPT://...` wouldn't match this

Reply:
Fixed.